### PR TITLE
Add font size commands to View menu

### PR DESCRIPTION
**Pull request by mnquintana**
Adds "Increase Font Size", "Decrease Font Size", and "Reset Font Size" to the View menu, which is typical in other editors (TextMate, Brackets) and should help in discoverability.

### CODE REVIEW

**Comment by izuzak** [comment]
What do you think about grouping these as sub-items in a `View > Font` menu? That's what TextMate does.

**Comment by mnquintana** [comment]
@izuzak I wouldn't mind that, but I do tend to dislike too many submenus, so I'm not sure.

**Comment by lee-dohm** [comment]
I think the View menu is compact enough that I could easily go either way on this one.

**Comment by mnquintana** [comment]
I think I'll leave it as-is for now, and if the View menu gets really long we can always move it to a Font submenu then.

File: menus/linux.cson [hunk 1]
```diff
@@ -125,6 +125,10 @@
         ]
       }
       { type: 'separator' }
+      { label: '&Increase Font Size', command: 'window:increase-font-size' }
+      { label: '&Decrease Font Size', command: 'window:decrease-font-size' }
+      { label: '&Reset Font Size', command: 'window:reset-font-size' }
```
Comment (izuzak) [inline menus/linux.cson]: Will this cause problems with this command in the same menu:

```
       { label: '&Reload', command: 'window:reload' }
```

We would now have two menu options starting with `&R...`

**Comment by izuzak** [comment]
@mnquintana I'm :cool: with that. Thanks :+1:

File: menus/linux.cson (hunk 1 above)
Comment (mnquintana) [inline menus/linux.cson]: Oh damn! Good catch, I didn't see that one. Fixing...

**Pull request merged**